src/components/CivicProfileForms/BasicInfo.jsx

```diff
@@ -1,8 +1,169 @@
+import FormControl from '@mui/material/FormControl';
+import FormHelperText from '@mui/material/FormHelperText';
+import Grid from '@mui/material/Grid';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import TextField from '@mui/material/TextField';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+// Hook Imports
+import { useCivicProfile } from '@hooks';
+// Component Imports
+import { FormSection } from '../Form';
 
-const BasicInfo = () => <Typography>Basic Info</Typography>;
+/**
+ * BasicInfo Component - A form to fill out basic user info
+ *
+ * @memberof CivicProfileForms
+ * @name BasicInfo
+ * @returns {React.JSX.Element} The BasicInfo Component
+ */
+const BasicInfo = () => {
+  const { data, add, isSuccess } = useCivicProfile();
+  const [formData, setFormData] = useState({
+    legalFirstName: '',
+    legalLastName: '',
+    legalDOB: null,
+    legalGender: ''
+  });
+
+  useEffect(() => {
+    if (isSuccess) {
+      setFormData((prevFormData) => ({ ...prevFormData, ...data }));
+    }
+  }, [isSuccess, data]);
+
+  const handleChange = (event) => {
+    if (event && event.target) {
+      const { name, value } = event.target;
+      setFormData((prevFormData) => ({ ...prevFormData, [name]: value }));
+    } else {
+      setFormData({
+        ...formData,
+        legalDOB: event
+      });
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (!isSuccess) {
+      return;
+    }
+    add(formData);
+  };
+
+  const handleClear = () => {
+    setFormData({ legalFirstName: '', legalLastName: '', legalDOB: '', legalGender: '' });
+  };
+
+  return (
+    <FormSection
+      title="Basic Information"
+      component="form"
+      onSubmit={handleSubmit}
+      autoComplete="off"
+    >
+      <Grid container spacing={{ xs: 2, sm: 2 }}>
+        <Grid item xs={12} sm={6}>
+          <TextField
+            id="hmis-basic-info-first-name"
+            name="legalFirstName"
+            label="Legal first name"
+            onChange={handleChange}
+            value={formData.legalFirstName}
+            fullWidth
+            autoFocus
+          />
+        </Grid>
+        <Grid item xs={12} sm={6}>
+          <TextField
+            id="hmis-basic-info-last-name"
+            name="legalLastName"
+            label="Legal last name"
+            onChange={handleChange}
+            value={formData.legalLastName}
+            fullWidth
+          />
+        </Grid>
+        <Grid item xs={12} sm={6}>
+          <FormControl fullWidth>
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                id="hmis-basic-info-date-of-birth"
+                name="legalDOB"
+                label="Date of birth"
```

Comment:
It think the issue with the test is that when <DatePicker /> renders to HTML, it does not add this label to the input element. Instead, it adds a default aria label, `choose date`. The test is grabbing this aria label for the input rather than the label name.

Reply:
This MUI issue has a fix for it:
https://github.com/mui/mui-x/issues/8146

Adding that fixes the datepicker failure, but reveals some issues in other parts of the test.

The gender select has a role of "combobox" not "textbox"

And the "submitButton" is failing because there are multiple buttons on the page.  Using "queryByRole" instead of "getByRole" will return all buttons on the page. That should help with getting the test to behave properly.

Reply:
I believe I'm making progress with the tests, but am still running into some. I've tried several different approaches but hesitate making any changes and so wanted to show what the current version is in the likely case I'm missing something.